starmap/specifications/primitives/gesture_recognizers/GestureRecognizer.md

```diff
@@ -50,7 +50,7 @@ How a gesture recognizer is associated with an element is a platform implementat
 
 A gesture recognizer should expose an API for adding objects that will receive state change events.
 
-These observers should be invoked each time the gesture recognizer's state value is written to.
+These observers should be notified each time the gesture recognizer's state changes.
```

Comment:
"value is written to" is intentional; the gesture might repeatedly set its "changed" state.